migration/sql-files/039-unique-area-name-create-new-area.sql

```diff
@@ -0,0 +1,25 @@
+------ You can't allow the same area name and the same ancestry inside a space
+
+ALTER TABLE areas ADD CONSTRAINT areas_name_space_id_path_unique UNIQUE(space_id,name,path);
+
+------  For existing spaces in production, which dont have a default area, create one.
+--
+-- 1. Get all spaces which have an area under it with the same name.
+-- 2. Get all spaces not in (1)
+-- 3. insert an 'area' for all such spaces in (2)
+
+INSERT INTO areas
+            (created_at,
+             updated_at,
+             name,
+             space_id)
+SELECT current_timestamp,
```

Comment:
Correct.
Updated it here : https://github.com/almighty/almighty-core/pull/913/files#diff-6492d13af50b66ca7c7bea0a3bfd3ea0

Tried my best to make the SQL readable ;)